README.md

```diff
@@ -0,0 +1,713 @@
+
+The `version` parameter (`string` or `number`) will accept the following values:
+
+- `v#`, where `v` is the abbreviation for "version" and `1` is the major release value.
+- `1`, where `1` is a numeric value or string.
+- Parsed values must be greater than zero and less than or equal to the current major release version.
+
+```js
+var diacritics = require("diacritics-transliterator");
+diacritics.setVersion("v1"); // "v1" returned
+```
+
+Invalid values will be ignored and the version will remain unchanged.
+
+The returned value will be a string with a leading "v" (for version) followed by the currently set version number.
+
+### diacritics.getVersion()
+
+Upon initialization, this function will always return the current major release version of the diacritics API.
+
+```js
+var diacritics = require("diacritics-transliterator");
+diacritics.getVersion(); // "v1" returned
+```
+
+The returned value will be a string with a leading "v" (for version) followed by the currently set version number.
+
+### diacritics.getLanguage(code)
+
+Returns the variant(s) metadata and data for the selected language code.
+
+The `code` parameter (type: `string`) will accept any of the following case insensitive values:
+
+- [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code (e.g. `de`)
+- The language written in English (e.g. `German`)
+- The language written in the native language (e.g. `Deutsch`)
+
+The returned value will be an object which includes the root language and any variants.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var german = diacritics.getLanguage("de");
+/* data =>
+{
+    "de": {
+        "metadata": { ... }
+        "data": { ... }
+    }
+}
+*/
+```
+
+If a language is not found, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getLanguage("test");
+// test => { "message": "Language 'test' was not found" }
+```
+
+### diacritics.getVariant(code)
+
+Returns the metadata and data for the selected language variant code.
+
+The `code` parameter (type: `string`) will accept any of the following case insensitive values:
+
+- language variant code (e.g. `de`)
+- The language variant name written in English.
+
+The returned value will be an object which includes only the variant language.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var german = diacritics.getVariant("de");
+/* german =>
+{
+    "de": {
+        "metadata": { ... }
+        "data": { ... }
+    }
+}
+*/
+```
+
+If a variant is not found, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getVariant("test");
+// test => { "message": "Variant 'test' was not found" }
+```
```

Comment:
In case of `language` you've specified `ISO 639-1`, but in a standard is missing for `variant`.